infra/app/service/identity_provider.tf

```diff
@@ -0,0 +1,54 @@
+locals {
```

Comment:
This change (eg. the whole file) is tangential to the main point of the PR

Reply:
wanna just move this change to a separate PR that we can merge quickly? seems like an easy one to pull out since it should just result in no-op terraform plans

Reply:
Made a ticket for this one https://github.com/navapbc/template-infra/issues/778